### DISCUSSION TITLE
Edge not deleted

### DISCUSSION THREAD

**Discussion by que762** [Atlas-OS/Atlas#1239] [Help & Support] [answered]
Hello everyone,

After I finished installing Atlas, I saw that Edge was still installed on my computer.
I tried to use the ‘Install or remove Edge’ option from the Atlas folder, but the system tells me that it has completed the operation even though Edge is still present.

Do you know if this is a bug in my Atlas installation?

Thanks for your answers!

**Comment by que762** [comment]
Update : I had to use the Uninstall option in the App Windows parameter to remove it.
But it's still strange that Atlas didn't do it itself...

**Reply by bert003** [reply]
When I tried to manually uninstall Edge, I lost the search functionality when pressing the Windows (start) button. When trying to search, a progress bar keeps displaying in rotation from left to right (and obviously no results). Did that happen to you?

**Comment by pencilnav** [comment, marked as answer]
Hi there, please use the new version of EdgeRemover to fix the issue.

Open powershell and paste:
```
iex(irm https://cdn.jsdelivr.net/gh/he3als/EdgeRemover@main/get.ps1)
```

**Comment by he3als** [comment]
The latest EdgeRemover is included in v0.4.1

**Reply by bert003** [reply]
I just executed 0.4.1 and still Edge is not removed on Windows 11 Pro.

**Comment by bert003** [comment]
Also when trying to execute "Install or Remove Edge.cmd" I get the below. Had to use revo uninstaller to remove edge since after the cmd script, edge is still there.

![edge](https://github.com/user-attachments/assets/05dc2e45-7e8d-482b-a869-6206a3aaff66)

**Reply by Abstractw1** [reply]
Yep same error here, can't uninstall edge

**Comment by Abstractw1** [comment]
Doesn't work, edge is not uninstalled and script doesn't work, pls fix
